src/context/node.js

```diff
@@ -218,6 +218,22 @@ export class ContextNode {
       setTimeout
     );
 
+    // Shadow root: track slot changes.
+    if (node.nodeType == FRAGMENT_NODE) {
```

Comment:
FMI: `FRAGMENT_NODE` means that its the root right?

Reply:
Curtesy of web spec, it means that it's a shadow root.